### DISCUSSION TITLE
Content types and format

### DISCUSSION THREAD

**Discussion by dpetka2001** [Goldenfreddy0703/Otaku#58] [Q&A] [answered]
Could someone give an explanations as to what `Set content types` and `Set content format` are 
in the options of the plugin? In what case should someone use them?

**Comment by Goldenfreddy0703** [comment, marked as answer]
Hey so Content types are basically viewtypes for the menu items and Content format is basically what Content you want displayed in the menu items. Example: I want my Content to just show anime movies.

I will be honest, it's kinda confusing so we will probably remove Content type or rename it to something else.

**Reply by dpetka2001** [reply]
I see, thanks for the clarification.
